gpslogger/build.gradle

```diff
@@ -149,6 +151,11 @@ dependencies {
         exclude group: 'com.google.android', module: 'android'
     }
 
+    //Android's WorkManager
+    implementation 'androidx.work:work-runtime:2.9.0'
+    // It needs Gson
+    implementation 'com.google.code.gson:gson:2.10.1'
```

Comment:
Gson needed as a simple way to serialize a class to a string and pass it as Data to the Worker.